src/basho_bench_duration.erl

```diff
@@ -94,6 +94,12 @@ handle_info(Msg, State) ->
 
 
 terminate(Reason, #state{duration=DurationMins}) ->
+    case Reason of
+        {shutdown, normal} ->
+            ?CONSOLE("Test completed after ~p mins.\n", [DurationMins]);
+        {shutdown, Reason} ->
+            ?CONSOLE("Test stopped: ~p\n", [Reason])
+    end,
```

Comment:
put `case Reason of ` statement in front of `supervisor:terminate_child(basho_bench_sup, basho_bench_run_sup)` or else it will never be executed.